src/boot/efi/stub.c

```diff
@@ -246,25 +255,22 @@ static EFI_STATUS load_addons_from_dir(
 
 }
 
-static EFI_STATUS cmdline_append_and_measure_addons(
-                EFI_HANDLE stub_image,
-                EFI_LOADED_IMAGE_PROTOCOL *loaded_image,
-                const char16_t *prefix,
-                const char *uname,
-                bool *ret_parameters_measured,
-                char16_t **cmdline_append) {
+static EFI_STATUS process_and_measure_addons(
+        EFI_HANDLE stub_image,
+        EFI_LOADED_IMAGE_PROTOCOL *loaded_image,
+        const char16_t *prefix,
+        const char *uname,
+        struct addon_additions *additions) {
```

Comment:
we usually indent parameter lists to double the usual indentation level, i.e. 16spaces, as before